DEPENDENCIES

```diff
@@ -169,7 +169,7 @@ npm/npmjs/-/cssom/0.3.8, MIT, approved, clearlydefined
 npm/npmjs/-/cssom/0.4.4, MIT, approved, clearlydefined
 npm/npmjs/-/cssstyle/2.3.0, MIT, approved, clearlydefined
 npm/npmjs/-/csstype/3.1.1, MIT, approved, clearlydefined
-npm/npmjs/-/cx-portal-shared-components/0.10.0, Apache-2.0 AND MIT AND OFL-1.1, approved, #7083
+npm/npmjs/-/cx-portal-shared-components/1.4.0, , restricted, clearlydefined
```

Comment:
please open an IP issue ticket for restricted dependencies using the dash tool according to the [guide](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04). If you don't have committers, let us know and we'll run that for you.

Reply:
Checking

Reply:
@ajithsimons Can we check on this.